enterprise-suite/es-monitor-api/default-monitors.json

```diff
@@ -104,6 +104,24 @@
           "description": "{{$labels.instance}} has anomalous scrape_duration_seconds"
         }
       },
+      "es_monitor_api_errors": {
+        "monitorVersion": "1",
+        "model": "threshold",
+        "parameters": {
+          "metric": "es_monitor_api_last_error",
+          "window": "1m",
+          "confidence": "1",
+          "severity": {
+            "warning": {
+              "comparator": ">",
+              "threshold": "0"
+            }
+          },
+          "summary": "es console monitor errors",
```

Comment:
"lightbend" instead of "es" seems more clear to me. Imagine this showing up in PagerDuty at 3am

Reply:
agree will change